fdbserver/DataDistributionTracker.actor.cpp

```diff
@@ -290,6 +297,32 @@ ACTOR Future<Void> trackShardBytes(
 	}
 }
 
+ACTOR Future<Void> readHotDetector(DataDistributionTracker* self) {
+	try {
+		loop {
+			state KeyRange keys = waitNext(self->readHotShard.getFuture());
+			state Transaction tr(self->cx);
+			loop {
+				try {
+					Standalone<VectorRef<KeyRangeRef>> readHotRanges = wait(tr.getReadHotRanges(keys));
+					for (auto& keyRange : readHotRanges) {
+						TraceEvent("ReadHotRangeLog")
```

Comment:
If there is a storm of read hot shards, we will get a lot of trace events. 
I think we should limit the number of trace events in a short period of time.

Also when the number of hot shards in a short period exceeds a threshold, we may want to output SevWarnAlways to alert DBA.

Reply:
The frequency, and actually the usefulness, of the log is yet to be determined after running some circus tests. For now, one can turn the knob `SHARD_READ_HOT_BANDWITH_MIN_PER_KSECONDS` to a large value to reduce/disable these logs.